integrations-and-supported-tools/sacred/notebooks/Neptune_Sacred.ipynb

```diff
@@ -247,145 +304,140 @@
+   ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
    "source": [
     "ex.run()"
-   ],
-   "outputs": [],
-   "metadata": {
-    "colab": {
-     "base_uri": "https://localhost:8080/",
-     "height": 194,
-     "referenced_widgets": [
-      "4ff2f58c889445a891e93fdbf4801832",
-      "1c4cbf83896a4aa884f72a151fefab7e",
-      "f6f628a577734f58afd93ba81c280376",
-      "b046bff22c3e4fb68cce386b977258b6",
-      "9a87108dd16f4b61b8fe34cbbf6a170a",
-      "df861e99463545d7ac84cebde39368a6",
-      "009779801ecd44afa2f524ae371c6b2f",
-      "1cfea757feb546e39fc69a80f53f86cb"
-     ]
-    },
-    "id": "_0CWFytQwapG",
-    "outputId": "9b4d8474-e341-479d-f10a-fccc123fba64"
-   }
+   ]
   },
   {
    "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
    "source": [
-    "## More Options"
+    "# More Options"
    ]
   },
   {
    "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
    "source": [
-    "### Log Artifacts\n",
+    "## Log Artifacts\n",
     "\n",
     "Model architecture and weights"
-   ],
-   "metadata": {
-    "id": "faU6X8eXXO3Z"
-   }
+   ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
    "source": [
     "model_fname = \"model\"\n",
     "print(f\"Saving model archictecture as {model_fname}.txt\")\n",
     "with open(f\"{model_fname}_arch.txt\", \"w\") as f:\n",
     "    f.write(str(model))"
-   ],
-   "outputs": [],
-   "metadata": {
-    "colab": {
-     "base_uri": "https://localhost:8080/"
-    },
-    "id": "3OB8Q_YD2tRi",
-    "outputId": "b0fd5c47-53ba-4fe8-a139-e4e72593d6b0"
-   }
+   ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
    "source": [
     "print(f\"Saving model weights as {model_fname}.pth\")\n",
     "torch.save(model.state_dict(), f\"./{model_fname}.pth\")"
-   ],
-   "outputs": [],
-   "metadata": {
-    "colab": {
-     "base_uri": "https://localhost:8080/"
-    },
-    "id": "Z4oRk7OW2zas",
-    "outputId": "5d949d6c-c23e-4553-86b6-532a8b7b3307"
-   }
+   ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
    "source": [
     "ex.add_artifact(filename=f\"./{model_fname}_arch.txt\", name=model_fname + \"_arch\")\n",
     "ex.add_artifact(filename=f\"./{model_fname}.pth\", name=model_fname)"
-   ],
-   "outputs": [],
-   "metadata": {
-    "id": "noBrWT7SXIBb"
-   }
+   ]
   },
   {
    "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
    "source": [
-    "## Stop logging\n",
+    "# Stop logging\n",
```

Comment:
intentional?

Reply:
yes, because this is needed for both the basic example and more options